### ISSUE TITLE
Docker old version of 'volume-enabled' is deprecated but the new version doesn't seem to work

### ISSUE THREAD

**Issue by Naxora** [kestra-io/kestra#3531]
### Describe the issue

TL;DR
The `kestra.tasks.scripts.docker.volume-enabled` is deprecated but the plugin configuration new version doesn't seem to work.

I got some help Slack how to setup plugin-configurations enablement of the 'volume-enabled'
[plugin-configurations](https://kestra.io/docs/configuration-guide/plugins#plugin-configurations-to-enable-or-disable-features)
But I run into issue.

My flow:
```yaml
tasks:
  - id: docker
    type: io.kestra.plugin.scripts.shell.Commands
    docker:
      image: alpine:latest
      volumes:
        - /path/to:/mnt
    commands:
      - cd /mnt
      - ls -la
```

The new way of 'volume-enabled' setting:
```yaml
        kestra:
          plugins:
            configurations: 
            - type: io.kestra.plugin.scripts.runner.docker.DockerTaskRunner
              values: 
                volume-enabled: true
```

Modification in docker-compose file in 'KESTRA_CONFIGURATION'
```yaml
[...]
    environment:
      KESTRA_CONFIGURATION: |
        datasources:
          postgres:
            url: jdbc:postgresql://postgres:5432/kestra
            driverClassName: org.postgresql.Driver
            username: kestra
            password: k3str4
        kestra:
          plugins:
            configurations: 
            - type: io.kestra.plugin.scripts.runner.docker.DockerTaskRunner
              values: 
                volume-enabled: true
          server:
            basic-auth:
              enabled: false
              username: "admin@kestra.io" # it must be a valid email address
              password: kestra
          repository:
            type: postgres
          storage:
            type: local
            local:
              base-path: "/app/storage"
          queue:
            type: postgres
          tasks:
            tmp-dir:
              path: /tmp/kestra-wd/tmp
          url: http://localhost:8080/
    ports:
    [...]
```

But it doesn't seem to work. The mapped folder is empty.
But if I use the deprecated version it works perfectly.
[deprecated version](https://kestra.io/docs/developer-guide/scripts/bind-mount)

### Environment

- Kestra Version: 0.16.0
- Operating System (OS/Docker/Kubernetes): Docker on linux server
- Java Version (if you don't run kestra in Docker): -

**Comment by anna-geller** [comment]
related https://github.com/kestra-io/kestra/issues/3608

**Comment by loicmathieu** [comment]
Fixed by https://github.com/kestra-io/kestra/pull/3726
